haskell/c2hs.bzl

```diff
@@ -8,7 +8,7 @@ load(
     "target_unique_name",
 )
 load(
-    ":private/providers.bzl",
+    "@io_tweag_rules_haskell//haskell:private/providers.bzl",
```

Comment:
Necessary?

Reply:
Could you add a note inline about this at the top of `providers.bzl` about this?